src/Quasar/Mount/SparkHDFS.purs

```diff
@@ -54,7 +54,7 @@ fromJSON ∷ Json → Either String Config
 fromJSON
   = fromString
   <=< (_ .? "connectionUri")
-  <=< (_ .? "spark")
```

Comment:
It should not - `spark://` is the format that quasar expects.

We should probably add property tests for the codecs!